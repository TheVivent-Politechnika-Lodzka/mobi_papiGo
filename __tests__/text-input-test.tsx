import 'react-native';
import React from 'react';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';
import TextInput from '../src/components/text-input';

it('Button render test', () => {
  const button = renderer
    .create(<TextInput value="" onChange={() => {}} />)
    .toJSON();
  expect(button).toMatchSnapshot();
});
