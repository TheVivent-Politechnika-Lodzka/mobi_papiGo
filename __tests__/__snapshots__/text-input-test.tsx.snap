// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button render test 1`] = `
<TextInput
  onChange={[Function]}
  style={
    Object {
      "backgroundColor": "#ddd",
      "borderColor": "gray",
      "borderRadius": 5,
      "borderWidth": 1,
      "fontSize": 20,
      "height": 60,
      "marginBottom": 10,
      "width": "90%",
    }
  }
  value=""
/>
`;
